src/util/ValidationUtil.js

```diff
@@ -0,0 +1,36 @@
+export class ValidationUtil {
+
+  static checkElementIsEmpty(value) {
+    return value == null || value.length === 0;
```

Comment:
No need to check all the conditions. It's enough only to cast `value` to boolean 
` ... return !!value `
**************
You can just check if the variable has a truthy value or not. That means
```
if( value ) {
}
```
will evaluate to true if value is not:

1. null
1 .undefined
1. NaN
1. empty string ("")
1. 0
1. false

The above list represents all possible false values in ECMA-/Javascript. Find it in the specification at the `ToBoolean` [section](http://www.ecma-international.org/publications/files/ECMA-ST/ECMA-262.pdf).